src/main/java/org/cactoos/http/HtWire.java

```diff
@@ -40,6 +45,12 @@
  * @version $Id$
  * @since 0.1
  */
+@SuppressWarnings(
+    {
+        "PMD.OnlyOneConstructorShouldDoInitialization",
+        "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"
```

Comment:
@krzyk Do you understand why PMD think you violate this rules?

Reply:
@SharplEr Yes, because those PMD checks were developed without lambdas in mind so they will always fail if there is any lambda in the constructor.

Reply:
@krzyk Thank you, I didn't know it.